numba/cuda/cudadrv/driver.py

```diff
@@ -233,8 +236,9 @@ def initialize(self):
             _logger.info('init')
             self.cuInit(0)
         except CudaAPIError as e:
-            self.initialization_error = e
-            raise CudaSupportError("Error at driver init: \n%s:" % e)
+            description = f"{e.msg} ({e.code})"
```

Comment:
Could this just be `str(e)`?

Reply:
I suppose so, but I thought writing the message first was a tiny bit nicer for the user, e.g.

```
CUDA_ERROR_UNKNOWN (999)
```

vs

```
[999] CUDA_ERROR_UNKNOWN
```

since the user tends to think in terms of the names of the errors rather than the error code numbers (which are an implementation detail to some extent).

Reply:
Agree, let's leave it.